APM/PSRCommandLogger.php

```diff
@@ -56,14 +61,24 @@ public function commandStarted(CommandStartedEvent $event)
             return;
         }
 
+        if ($this->stopwatch !== null) {
+            $this->stopwatch->start('mongodb', 'mongodb');
```

Comment:
I would suggest using the command name here to make the data more informative. Same in the corresponding `stop` calls.

Reply:
Maybe with a `mongodb_` prefix?

It looks like this right now:
![image](https://user-images.githubusercontent.com/4293906/67100341-c48af780-f1bf-11e9-8af6-972a679660a9.png)